### PR TITLE
Allow popover links to open on right clicks

### PR DESCRIPTION
**Pull request by embooglement**
This commit adds two new parameters to the `PopoverLinkComponent`, `openOnLeftClick` and `openOnRightClick`. Setting either of these to true will open the popover on the corresponding click event. The default behavior is to only open on left-clicks.

### CODE REVIEW

**Comment by embooglement** [comment]
@Addepar/fire @Addepar/pants

File: src/popover_link.coffee [hunk 1]
```diff
@@ -44,4 +46,18 @@ Ember.Widgets.PopoverLinkComponent = Ember.Component.extend
...
       popover.appendTo @get('rootElement')
 
+  click: ->
+    if not @get('openOnLeftClick')
+      return false
+
+    @_openPopover()
+    return false
+
+  contextMenu: ->
```
Comment (angelagu) [inline src/popover_link.coffee]: This should be renamed to `openContextMenu` instead

Comment (embooglement) [inline src/popover_link.coffee]: `contextMenu` is the name of the action, this can't be changed.

Comment (embooglement) [inline src/popover_link.coffee]: Similar to `click`

**Comment by angelagu** [comment]
Why would we want to disable left or right click?

**Comment by angelagu** [comment]
lgtm

**Comment by josheisner** [comment]
Is this what I think it is??? :o

File: src/popover_link.coffee (hunk 1 above)
Comment (cyril-sf) [inline src/popover_link.coffee]: http://emberjs.com/api/classes/Ember.View.html#toc_event-names

**Pull request merged**